packages/reactive-element/src/reactive-element.ts

```diff
@@ -490,8 +490,9 @@ export abstract class ReactiveElement
    * @nocollapse
    */
   static addInitializer(initializer: Initializer) {
+    this.finalize();
     this._initializers ??= [];
-    this._initializers.push(initializer);
+    this._initializers!.push(initializer);
```

Comment:
🏌️‍♂️: `(this._initializers ??= []).push(initializer)`

Reply:
Done.